File: Python/program68.py
class Digit:
    _No = 0
    
    def __init__(self, iNo):
        self._No = iNo
           
    def Count_digit(self):
        
        if(self._No < 0):
            self._No = -self._No
        iCount = 0
        while(self._No > 0):
            iDigit = self._No % 10
            iCount += 1
            self._No = self._No // 10
            
        return iCount
    
def main(): 
    
    iValue = int (input("Enter number : "))
        
    sobj = Digit(iValue)
    iRet = sobj.Count_digit()
    
    print("Number of digits are  ",iRet)

if __name__ == "__main__":
    main()

File: C/program68.c


#include<stdio.h>

int CountFrequency(int iNo)
{
    if(iNo < 0)
    {
        iNo = - iNo;
    }
    
    int iCount =0, iDigit = 0;

    while(iNo > 0)
    {   
        iDigit = iNo % 10;
        if(iDigit == 5)
        {
            iCount++;
        }
     
        iNo = iNo / 10;
        
    }
    
    return iCount;
}

int main()
{
    int iValue = 0, iRet = 0;

    printf("Enter number :\n");
    scanf("%d",&iValue);

    iRet = CountFrequency(iValue);

    printf("The frequency of 5 is : %d\n",iRet);

    return 0;
}